ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -770,5 +770,22 @@
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/remove-managing-organisation/about-to-submit",
     "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/remove-managing-organisation/submitted",
     "EventEnablingCondition": "outsourcingPolicy.Organisation.OrganisationID != \"\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "addGatekeepingOrder",
+    "Name": "V2: Add the gatekeeping order",
```

Comment:
Need to remove V2 from the name here

Reply:
will do :) wasn't sure as it was like this in protoype